scrapers/Xvideos.yml

```diff
@@ -1,10 +1,29 @@
-name: xvideos
+name: Xvideos
 sceneByURL:
   - action: scrapeXPath
     url:
       - xvideos.com
+      - xvideos2.com
+      - xvideos.es
```

Comment:
I would prefer to exclude this from the scraper. It's machine translated metadata, not original releases, which would create confusion if submited to the stash-box instances.

Reply:
If a user uses one of those domains, he would be left off by Stash. 

- xvideos2.com
This domain redirects to xvideos.com. See: https://wheregoes.com/trace/2023732209/
yt-dlp also supports this domain.

- xvideos.es
_"create confusion if submited to the stash-box instances."_
stash-box may implement some mecanism (tag aliases, rejection of text written in an unknown language etc) allowing only metada in a specific language.

I would exclude xvideos.es only. Do you agree ?

Reply:
I only meant `xvideos.es`, not sure why it selected both. Sorry for the confusion.

stash-box is just the software so not sure it will implement any language restrictions. But canonical community instance StashDB allows non-English content via unique entry as long as it has an official link. And I think this would create confusion as they are not official releases, but just AI translated metadata of the English release. 

From Stash perspective I can see the value in them and managing them is up to individual user. Maybe instead of removing it, you can just comment out so it's not used by default.

Reply:
Comment done.